src/tracker/tracker-funcs.jl

```diff
@@ -221,23 +221,25 @@ end
     get_large_floe_condition(
     area1,
     ratios,
+    large_floe_minimum_area,
     large_floe_settings=(
         area=1200, arearatio=0.28, majaxisratio=0.10, minaxisratio=0.12, convex_area=0.14
     ),
 )
 
-Set of conditions for "large" floes. Return `true` if the area of the floe is greater than `large_floe_settings.area` and the similarity ratios are less than the corresponding thresholds in `large_floe_settings`. Return `false` otherwise. Used to determine whether to call `match_corr`.
+Set of conditions for "large" floes. Return `true` if the area of the floe is greater than `large_floe_minimum_area` and the similarity ratios are less than the corresponding thresholds in `large_floe_settings`. Return `false` otherwise. Used to determine whether to call `match_corr`.
 
 See also [`get_small_floe_condition`](@ref).
 """
 function get_large_floe_condition(
     area1,
     ratios,
+    large_floe_minimum_area = 1200,
     large_floe_settings=(
-        area=1200, arearatio=0.28, majaxisratio=0.10, minaxisratio=0.12, convex_area=0.14
+        arearatio=0.28, majaxisratio=0.10, minaxisratio=0.12, convex_area=0.14
     ),
 )
-    return area1 > large_floe_settings.area &&
+    return area1 > large_floe_minimum_area &&
```

Comment:
Shouldn't this be 
```suggestion
    return area1 >= large_floe_minimum_area &&
```
and the other be `<`? If I'm at the minimum area, then I fulfill the condition. If I'm smaller than it, I don't. So This should be greater than or equal to.

Reply:
👍